data/json/monsters/mammal.json

```diff
@@ -2516,42 +2522,28 @@
     "type": "MONSTER",
     "name": { "str": "domestic rabbit kit" },
     "description": "A tiny bunny with a cute wiggling nose, a cotton ball tail, and a body made of delicious flesh.",
-    "copy-from": "mon_rabbit_base",
-    "volume": "150 ml",
-    "weight": "30 g",
-    "hp": 1,
-    "speed": 125,
-    "attack_cost": 125,
-    "dodge": 8,
+    "copy-from": "mon_rabbit_baby_base",
     "upgrades": { "age_grow": 45, "into": "mon_rabbit" },
-    "petfood": { "food": [ "CATTLEFOOD" ], "feed": "The %s seems to like you!  It lets you pat its head and seems friendly." },
-    "harvest": "mammal_tiny"
+    "petfood": { "food": [ "CATTLEFOOD" ], "feed": "The %s seems to like you!  It lets you pat its head and seems friendly." }
   },
   {
     "id": "mon_wild_rabbit",
     "type": "MONSTER",
     "name": { "str": "cottontail rabbit" },
-    "description": "A brown-coated cottontail rabbit, with a fluffy white tail.  Its small size and docile nature makes it easy prey for predators, at least the ones that can catch it.",
+    "description": "A brown-coated cottontail rabbit, with a fluffy white tail.  Its small size and docile nature makes it easy prey for predators, at least the ones that can catch it.  You can't tell which one of the two cottontail species from the region this is.",
     "copy-from": "mon_rabbit_base",
     "looks_like": "mon_rabbit",
+    "weight": "1200 g",
     "reproduction": { "baby_monster": "mon_wild_rabbit_baby", "baby_count": 5, "baby_timer": 28 }
   },
   {
     "id": "mon_wild_rabbit_baby",
     "type": "MONSTER",
     "name": { "str": "cottontail rabbit kit" },
     "description": "A tiny and very cute cottontail rabbit kit.",
-    "copy-from": "mon_rabbit_base",
+    "copy-from": "mon_rabbit_baby_base",
     "looks_like": "mon_rabbit_baby",
-    "volume": "150 ml",
-    "weight": "30 g",
-    "hp": 1,
-    "speed": 175,
-    "attack_cost": 175,
-    "dodge": 8,
-    "//": "Sexually mature at 2-3 months. Domestic rabbits are 4-5 months and have a value of 45",
-    "upgrades": { "age_grow": 27, "into": "mon_wild_rabbit" },
-    "harvest": "mammal_tiny"
+    "upgrades": { "age_grow": 23, "into": "mon_wild_rabbit" }
```

Comment:
It might be nice to leave around the comment on how this was calculated, because it's not clear to me where the numbers for any of these come from.

Reply:
It's basically days divided by 4. This is more of a matter of our documentation regarding monster evolution factor than anything here.

Reply:
Oh, I forgot about evolution. You're right, that doesn't need documentation.